app/assets/javascripts/survey_modal.js.erb

```diff
@@ -0,0 +1,38 @@
+/* behavior for survey popup modal */
+
+setTimeout(() => {
+    $('#survey_modal').modal({
+        backdrop: false
+    })
+    $('body').removeClass('modal-open'); // allows background to be scrollable
+    $(document).off('focusin.modal'); // allows background form elements to be clickable
+}, 3000);
+
+function setSurveySeenCookie(expires) {
+    console.log('setSurveySeenCookie firing');
+    if (expires == 'max') {
+        var cookieExpDate = (new Date(Date.now() + 14 * 24 * 60 * 60 * 1000)).toUTCString();
+    } else if (expires == 'mid') {
+        var cookieExpDate = (new Date(Date.now() + 7 * 24 * 60 * 60 * 1000)).toUTCString();
+    } else {
+        var cookieExpDate = (new Date(Date.now() + 60 * 60 * 1000)).toUTCString();
+    }
+    document.cookie = '<%= ApplicationController.helpers.survey_cookie_name %>' + '=1;expires=' + cookieExpDate + ';path=/;Secure;SameSite=Lax';
+    console.log('setSurveySeenCookie cookie set!');
+}
+
+$('#survey_modal_agree').on('click', function() {
+    setSurveySeenCookie('max');
+    $('#survey_modal').modal('hide');
+});
+$('#survey_modal_later').on('click', function() {
+    setSurveySeenCookie('min');
+});
+$('#survey_modal_decline').on('click', function() {
+    setSurveySeenCookie('mid');
+});
+$('#survey_modal').on('hide.bs.modal', function() {
+    setSurveySeenCookie('min');
+});
+// for some reason syntax below causes setSurveySeenCookie to fire before click ??
+//$('#survey_modal_later').click(setSurveySeenCookie('min'));
```

Comment:
what happens if you just call `.click()?` or if you this?

```$('#survey_modal_later').click(function() {
  setSurveySeenCookie('min')
});```

Do you still get the same syntax error?